### PR TITLE
Adding new API version (2017-05-15-preview). No other changes made

### PR DESCRIPTION
**Pull request by finiteattractor**
Adding new API version (2017-05-15-preview). No other changes made, just copied all files and updated the version string.

### CODE REVIEW

**Review by viveklingaiah — CHANGES_REQUESTED** [review]
I think you will need to another block for api 2017-05-15-preview like this file has for 2015
https://github.com/AzureAutomationTeam/azure-rest-api-specs/blob/current/specification/automation/resource-manager/readme.md

so, that you can pass an argument for 2017 and generates SDK for you. Could you also try generating SDK using autorest. Weijie did it last  week, please check with him for commands for SDK generation and linter (static validation)

**Comment by safeermohammed** [comment]
@finiteattractor we can close this PR as this is not needed

File: specification/automation/resource-manager/Microsoft.Automation/2015-10-31/definitions.json [hunk 1]
```diff
@@ -2954,6 +2954,21 @@
       },
       "description": "The runbook property associated with the entity."
     },
+    "scheduleFrequency": {
+      "type": "string",
+      "description": "Gets or sets the frequency of the schedule.",
```
Comment (safeermohammed) [inline specification/automation/resource-manager/Microsoft.Automation/2015-10-31/definitions.json:2959]: In last review Azure team were against using gets and sets.  Can you change the description

**Review by safeermohammed — APPROVED** [review]

File: specification/automation/resource-manager/Microsoft.Automation/2015-10-31/definitions.json (hunk 1 above)
Comment (finiteattractor) [reply specification/automation/resource-manager/Microsoft.Automation/2015-10-31/definitions.json:2959]: updated all! Thanks!

**Comment by viveklingaiah** [comment]
Hi Mo, 
   Make sure you do not have descriptions starting with "gets or sets" for definitions, they need to be more descriptive and useful. Swagger team will block, otherwise. I believe you said you've already corrected ? Also, could you replace real subscription ids in examples with some dummy id. or "subid" or all zeros ? Swagger team will raise concerns for that as well. refer to an example for accounts or runbook

**Comment by viveklingaiah** [comment]
I can approve right now or you can change and then I approve.
let me know

**Review by viveklingaiah — APPROVED** [review]

**Pull request merged**